File: leetcode3/최원준/3898. Find the Degree of Each Vertex.py
#

'''
1. 아이디어 :
-

2. 시간복잡도 :
    O(n)

3. 자료구조/알고리즘 :
-

'''

class Solution:
    def findDegrees(self, matrix: list[list[int]]) -> list[int]:
        return [sum(m) for m in matrix]

File: leetcode3/염혜정/3898. Find the Degree of Each Vertex.java
// 완전탐색
// o(n^2)

class Solution {
    public int[] findDegrees(int[][] matrix) {
        int[] result = new int[matrix.length];
        for (int i = 0; i<matrix.length; i++) {
            for (int k = 0; k<matrix[0].length; k++) {
                if (matrix[i][k] == 1) result[i]++;
            }
        }
        return result;
    }
}
